.github/workflows/delete-data.yml

```
@@ -0,0 +1,29 @@
+name: '* Delete test data'
+
+on:
+  workflow_dispatch:
+    inputs:
+      project:
+        description: Delete data for a specific offender
+        type: choice
+        required: true
+        options:
+          - delete-previous-addresses-for-offender
+      crn:
+        description: Offender's CRN
+        type: string
+        required: true
+
+jobs:
+  delete-for-project:
+    uses: ./.github/workflows/test.yml
+    if: inputs.project == ' delete-previous-addresses-for-offender'
```

Is this extra space supposed to be there?
```suggestion
    if: inputs.project == 'delete-previous-addresses-for-offender'
```

---

good spot, removed.